build_debian.sh

```diff
@@ -445,6 +445,10 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'docker
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install gcc libpython2.7-dev
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'netifaces==0.10.7'
 
+# Get package to support Dynamic Port Breakout
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install xmltodict==0.12.0
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install jsondiff==1.2.0
```

Comment:
can you put those dependency in the sonic utilities so that we can automatically install them.

Reply:
Unfortunately, we are currently building sonic-utilities as a .deb package, so it cannot automatically install Python packages which are only available as .whls. We have encountered this limitation with other packages. See https://github.com/Azure/sonic-utilities/blob/master/setup.py#L140

Reply:
Marking it resolved. Kindly approve and merge.

Reply:
Unresolving to wait for @lguohan to re-review.

Reply:
let's book an item to convert the sonic-utilties into python wheel package?

Reply:
Can do. @lguohan: Do you believe these lines belong here or in sonic_debian_extension.j2?

Reply:
Moved to sonic_debian_extension.j2, Marking it resolved. Kindly approve and merge.